keras/src/print.py

```diff
@@ -0,0 +1,25 @@
+from keras.src.api_export import keras_export
+from keras.src.backend import backend as keras_backend
+
+_print = print
+
+
+@keras_export("keras.print")
```

Comment:
Instead, please add individual print ops to e.g. `keras/src/backend/tensorflow/core.py`, `keras/src/backend/jax/core.py`, etc. Then export the op is `keras/src/ops/core.py`.

Reply:
Ok I'm not sure if you want `from jax.debug import print` and `from tensorflow import print` or if you want them with first-class docstrings. I've added the latter; and updated this PR.

Reply:
I believe the request was to add a `print` implementation also for the torch, numpy and openvino backends (even if they're the same).

Then remove the fallback mechanism and `_print`.